lnwallet/channel.go

```diff
@@ -2120,6 +2120,8 @@ func (lc *LightningChannel) restorePendingLocalUpdates(
 	// If we did have a dangling commit, then we'll examine which updates
 	// we included in that state and re-insert them into our update log.
 	for _, logUpdate := range pendingRemoteCommitDiff.LogUpdates {
+		logUpdate := logUpdate
```

Comment:
Wow thought we fixed an instance of this in the past....

This _really_ should be a `go vet` level warning at this point....

Reply:
Pretty sure we did. The linter only warned once I changed a line around it :/

Reply:
Maybe re-run the `golang-cilint` on all of `lnd` without the `new-from-rev` flag and see if there is more.